src/sentiment-analysis.js

```diff
@@ -0,0 +1,19 @@
+/* This file contains the code for analyzing blog posts text to identify the
+ * negative or positve words being used in a post and return a summary of it
+ * along with a score. The file uses a node module called sentiment to implement
+ * the functionality of analyzing text of blogs. The function accepts striped
```

Comment:
Typo at: "The function accepts striped" should be "The function accepts stripped"
Otherwise looks good 👍 , need to rebase to match the current master branch.

Reply:
Hey @ImmutableBox ,
Resolved this in commit https://github.com/Seneca-CDOT/telescope/pull/46/commits/259b31c11f926423304437065cecd538abd8b11e